<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Detector</string>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListView" name="listView">
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="last_button">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>上一张</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="detect_button">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>检测</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="next_button">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>下一张</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="open_file"/>
    <addaction name="open_folder"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="edit_params"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="menu"/>
  </widget>
  <action name="open_file">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="open_folder">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="edit_params">
   <property name="text">
    <string>参数</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>detect_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>detect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>last_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>last()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_file</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_folder</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_folder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>select(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_params</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>edit_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>last()</slot>
  <slot>next()</slot>
  <slot>detect()</slot>
  <slot>open_file()</slot>
  <slot>open_folder()</slot>
  <slot>select(QModelIndex)</slot>
  <slot>edit_params()</slot>
 </slots>
</ui>
